doc/index.rst

```diff
@@ -142,13 +142,24 @@ this::
         V
     mailcow user management
 
-Purging Old Accounts
-++++++++++++++++++++
+Initializing the Bot Interface
+++++++++++++++++++++++++++++++
 
-The ``mailadm prune`` command will remove accounts of expired users. You should
-add a cron job which executes this once an hour, for example::
+You don't have to login with SSH every time you want to create tokens. You can
+also use the bot interface to talk to the bot in a verified Delta group. The
+bot needs an e-mail account to operate - it doesn't have to be on your mailcow
+server, it can be any e-mail account which also works with Delta Chat.
```

Comment:
```suggestion
You don't have to login with SSH or user docker commands every time you want to create tokens. You can
rather setup a dedicated bot e-mail address and bootstrap a verified "admin group chat", in order to add/remove/list tokens for e-mail account creation.  The bot's e-mail address can be any e-mail address and does not need to reside on the e-mail domain whose accounts it is managing. 
```

Reply:
Hm, not sure about that wording, but I rephrased it a bit along those lines. As it always is with writing, too many cooks...